src/loop_yml.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
why are you pushing this?

Reply:
It was a useful little code to run all the experiment files in config folder.

It's not essential so you can remove it you want

Reply:
could you remove it please?